applications/nrf_desktop/configuration/common/led_effect.h

```diff
@@ -132,8 +226,17 @@ struct led_effect {
 			.substep_time = LED_CLOCK_BLINK_PERIOD,		\
 		},							\
 
-/* UTIL_LISTIFY accepts just one additional argument - period is defined
+
+/** Create LED clock effect initializer.
+ *
+ * LED blinks a defined number of times, then it is turned off for a defined
+ * period of time. The sequence is repeated periodically.
+ *
+ * @note UTIL_LISTIFY accepts just one additional argument - period is defined
```

Comment:
@MarekPieta , please update this line as discussed f2f.

Reply:
done